kornia/geometry/epipolar/_metrics.py

```diff
@@ -99,12 +100,13 @@ def symmetrical_epipolar_distance(
     line2_in_1: Tensor = pts2 @ Fm
 
     # numerator = (x'^T F x) ** 2
-    numerator: Tensor = (pts2 * line1_in_2).sum(dim=-1).pow(2)
```

Comment:
Ca you include this in the Kornia core whitelist ? https://github.com/kornia/kornia/blob/master/kornia/core/_backend.py

Reply:
if einsum diverges a lot from numpy i would discard then. One thought is to "numpy ready" code, at some point we would like to support numpy as well